doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -89,10 +89,11 @@ These core principles will guide NFV verification deliverables:
 <a name="1.4.1"></a>
 ### 1.4.1 Overarching Objectives and Goals
 
-1. Deliver verified implementation of reference architecture which satisfies infrastructure needs for VNF-developer teams.
-2. All accomplished with augmentation to the current OVP ecosystem.
-3. Increase probability VNFs will on-board and function with minimal problems, or issues, during initial instantiation of VNF.
-4. Test Harnesses will be portable, or compatible, across all RAs/Distributions which already conform to standard interfaces and services.
+1. Deliver tools and methods for verifying NFVi capabilities satisfy infrastructure needs of VNFs identified in RM.
+2. Augment current OVP program to meet infrastructure verification requirements.
+3. Increase probability VNFs will on-board and function with minimal problems, or issues, during initial deployment of VNF.
+4. Increase confidence that VNF performance and capacity can be forecast for given NFVi
+5. Verification tools to be portable, or compatible, across all RAs/Distributions which already conform to standard interfaces and services.
```

Comment:
- Use uppercase "I" in NFVI throughout the document/chapter.  
- For no. 2, this should include xNF (VNF, CNF) in additional to NFVI?  Perhaps change the statement, "..meet infrastructure verification.." to "..meet infrastructure and network function verification.."?
- Do you think we take the time now to change all occurrences of "VNFs" to "VNFs/CNFs", or something equivalent, to give us flexibility to easily refactor our statements when CNFs are more prevalent, vs. having to modify at a later date?

Reply:
> * Do you think we take the time now to change all occurrences of "VNFs" to "VNFs/CNFs"
I think that makes sense at the RM level, where it is a document that is expressly intended to cover both VNFs and CNFs at the same time.  RA1 and RA2 diverge to focus on VNFs and CNFs, respectively, and I think that RA1->RI1 -> RC1 should use VNFs (to be crystal clear about the domain on which they are focused) and that an RA2/RI2/RC2 suite would be similarly focused on CNFs.

Reply:
@kedmison - agree that the RM level, and valid point re: keeping domains separate.  Therefore, removing last two comments, as NA.